examples/simpleGov/README.md

```diff
@@ -0,0 +1,1064 @@
+
+### Wire
+
+The `wire.go` file allows developers to register the concrete message types of their module into the codec. In our case, we have two messages to declare:
+
+```go
+func RegisterWire(cdc *wire.Codec) {
+    cdc.RegisterConcrete(SubmitProposalMsg{}, "simple_governance/SubmitProposalMsg", nil)
+    cdc.RegisterConcrete(VoteMsg{}, "simple_governance/VoteMsg", nil)
+}
+```
+Don't forget to call this function in `app.go` (see [Application - Bridging it all together](#application_-_bridging_it_all_together) for more).
+
+### Errors
+
+The `error.go` file allows us to define custom error messages for our module.  Declaring errors should be relatively similar in all modules. You can look in the [error.go](./error.go) file of our simple governance module for a concrete example. The code is self-explanatory.
+
+Note that the errors of our module inherit from the `sdk.Error` interface and therefore possess the method `Result()`. This method is useful when there is an error in the `handler` and an error has to be returned in place of an actual result.
+
+### Command-Line Interface and Rest API
+
+Each module can define a set of commands for the Command-Line Interface and endpoints for the REST API. Let us create a `client` repository to define the commands and endpoints for our simple governance module.
+
+```bash
+mkdir client
+cd client
+mkdir cli
+mkdir rest
+```
+
+#### Command-Line Interface (CLI)
+
+Go in the `cli` folder and create a `simple_governance.go` file. This is where we will define the commands for our module.
+
+The CLI builds on top of [Cobra](https://github.com/spf13/cobra). Here is the schema to build a command on top of Cobra:
+
+```go
+    // Declare flags
+    const(
+        Flag = "flag"
+        ...
+    )
+
+    // Main command function. One function for each command.
+    func Command(codec *wire.Codec) *cobra.Command {
+        // Create the command to return
+        command := &cobra.Command{
+            Use: "actual command",
+            Short: "Short description",
+            Run: func(cmd *cobra.Command, args []string) error {
+                // Actual function to run when command is used
+            },
+        }
+
+        // Add flags to the command
+        command.Flags().<Type>(FlagNameConstant, <example_value>, "<Description>")
+
+        return command
+    }
+```
+
+For a detailed implementation of the commands of the simple governance module, click [here](../client/cli/simple_governance.go).
+
+#### Rest API
+
+The Rest Server, also called [Light-Client Daemon (LCD)](https://github.com/cosmos/cosmos-sdk/tree/master/client/lcd), provides support for **HTTP queries**.
```

Comment:
This naming is confusing. I thought we were talking about the cosmos `lcd` not the simple_gov `lcd` may want better naming here.

Reply:
Hmm, this is just a general intro sentence to the LCD, nothing more